frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -124,12 +164,8 @@ export const TicketDetailsCard = () => {
 				</HeaderButtons>
 			</CardHeader>
 			<FormProvider {...formData}>
-				<Form>
-					<TicketForm template={template} ticket={ticket} />
-				</Form>
+				<TicketForm template={template} ticket={ticket} onBlur={onBlurHandler} />
```

Comment:
Should this be name `onPropertyBlur` instead, so that it's clearer that it refers to a property blurring rather than the whole form?